stdpopsim/genetic_maps.py

```diff
@@ -13,7 +14,7 @@
 class GeneticMap:
     """
     Class representing a genetic map for a species. Provides functionality for
-    downloading and cacheing recombination maps from a remote URL.
+    downloading and cacheing recombination/genetic maps from a remote URL.
```

Comment:
According to the docs above, "genetic map" means a crossing-over map, which is what this is, so no need for the "recombination/genetic" map?
```suggestion
    downloading and cacheing genetic maps from a remote URL.
```